src/inspectors/package/CPPPackageInspector.ts

```diff
@@ -0,0 +1,177 @@
+                throw ErrorFactory.newInternalError(`xml2js failed to parse pom.xml: ${err.message}`);
+            }
+            const xmlDependencies = result.project.dependencies.values();
+            for (const xmlDependency of xmlDependencies) {
+                const dependencyAttributes = xmlDependency.dependency.values();
+                for (const attribute of dependencyAttributes) {
+                    const packageName = `${attribute.groupId.pop()}:${attribute.artifactId.pop()}`;
+                    const version = attribute.version ? String(attribute.version.pop()) : undefined;
+                    this.parsedDependencies.push({ packageName, version });
+                }
+            }
+        });
+    }
+
+    private async resolveGradleFileString(gradleFileString: string) {
+        await g2js.parseText(gradleFileString).then((result: BuildGradle) => {
+            for (const dependency of result.dependencies) {
+                if (dependency.name.startsWith("'") && dependency.name.endsWith("'")) {
+                    dependency.name = dependency.name.slice(1, -1);
+                }
+                const version = dependency.version ? dependency.version : undefined;
+                this.parsedDependencies.push({ packageName: dependency.name, version });
+            }
+        });
+    }
+}
+
+export interface PomXML {
+    project: {
+        $: {
+            xmlns: string;
+            'xmlns:xsi': string;
+            'xsi:schemaLocation': string;
+        };
+        modelVersion: string;
+        parent: Record<string, unknown>;
+        groupId: string;
+        artifactId: string;
+        version: string;
+        name: string;
+        description: string;
+        properties: Record<string, unknown>;
+        dependencies: [
+            {
+                dependency: [
+                    {
+                        groupId: [string];
+                        artifactId: [string];
+                        version: [string];
+                        scope?: [string];
+                        exclusions?: [string];
+                    },
+                ];
+            },
+        ];
+        build: [
+            {
+                plugins: [
+                    {
+                        plugin: [
+                            {
+                                groupId: [string];
+                                artifactId: [string];
+                            },
+                        ];
+                    },
+                ];
+            },
+        ];
+    };
+}
+
+export interface BuildGradle {
+    plugins: [];
+    group: string;
+    version: string;
+    sourceCompatibility: string;
+    repositories: [];
+    dependencies: [
+        {
+            group: string;
+            name: string;
+            version: string;
+            type: string;
+            excludes: [];
+        },
+    ];
+}
+
+interface ParsedDependency {
+    packageName: string;
+    version: string | undefined;
+}
```

Comment:
I don't understand why did you just copy [JavaPackageInspector](https://github.com/DXHeroes/dx-scanner/blob/master/src/inspectors/package/JavaPackageInspector.ts) and only change the names. Why did you do it?